.rubocop.yml

```diff
@@ -0,0 +1,34 @@
+AllCops:
+  Exclude:
+    - "bin/*"
+    - "db/*"
+    - "config/*"
+    - "Guardfile"
+    - "Rakefile"
+  DisplayCopNames: true
+
+Metrics/LineLength:
```

Comment:
- [ ] There is no `Metrics/LineLength` rule. It's `Layout/LineLength`. Check it [here](https://rubocop.readthedocs.io/en/latest/cops_layout/#layoutlinelength)